### DISCUSSION TITLE
Apply for NGI Zero Entrust fund

### DISCUSSION THREAD

**Discussion by hb0nd** [organicmaps/organicmaps#4111] [Ideas]
https://nlnet.nl/entrust/

It's an EU programme that funds development of FOSS software. It previously funded StreetComplete, and is currently funding development for MapComplete & Nominatim, among others.

> We are seeking project proposals between 5 000 and 50 000 euros — with the possibility to scale them up if there is proven potential. Reliability, confidentiality, integrity, security and data portability should be the 'new normal' of the internet, something ordinary users should not have to worry about — users should be in control. So let's make it happen.

According to Mateusz Konieczny (StreetComplete dev & OSMF board member):
> Paperwork is not outrageous so if you have OSM projects limited by funds or time it may be a good idea to apply (feel free to ask for details, I went through this process and the largest problem was "how should I tax this income").

The third call of NGI Zero Entrust opened on December 1st 2022, with a deadline of February 1st 2023

**Comment by matkoniecz** [comment]
> It's an EU programme that funds development of FOSS software. It previously funded StreetComplete

Strictly speaking StreetComplete was funded from a bit different branch ( https://nlnet.nl/discovery/ ) but entrust is more fitting OSM ecosystem in general if anything.

**Comment by matkoniecz** [comment]
Oh, if anyone is from Germany then https://prototypefund.de/ may be also useful.

**Reply by biodranik** [reply]
Contributors from Germany are welcome to take this grant to develop something for OM, we would be glad to help!
